src/Workspaces/Core/Portable/Remote/RemoteServiceName.cs

```diff
@@ -22,6 +22,9 @@ namespace Microsoft.CodeAnalysis.Remote
         internal const string Prefix = "roslyn";
         internal const string IntelliCodeServiceName = "pythia";
         internal const string RazorServiceName = "razorLanguageService";
+        internal const string TestingAnalysisServiceName = "remoteTestDiscoveryHostService";
```

Comment:
Please rename to UnitTestingAnalysisServiceName. Also I think we decided to name this "remoteAnalysisService" on the testimpact side. Tagging @Cosifne to rename this appropriately on testimpact side.

Reply:
Should it be 'remoteTestAnalysisService' or 'remoteAnalysisService' ? I feel remote AnalysisService might be a little bit confusing in Roslyn because they have a lot of AnalysisService

Reply:
It looks like it is only tied to the json file's name, so it might be OK to call it remoteAnalysisService

Reply:
Isn't `remoteAnalysisService` to generic? The name should be specific to unit testing. Also doesn't need to include "remote".